frontend/javascripts/test/backend-snapshot-tests/__snapshots__/datasets.e2e.ts.snap

```diff
@@ -1,5 +1,33 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`Dataset API (E2E) > Dataset Paths 1`] = `
+[
+  {
+    "layerName": "segmentation",
+    "magLinkInfos": [
+      {
+        "linkedMags": [],
+        "mag": {
+          "dataLayerName": "segmentation",
+          "dataSourceId": {
+            "name": "test-dataset",
+            "team": "Organization_X",
+          },
+          "hasLocalData": true,
+          "mag": [
+            1,
+            1,
+            1,
+          ],
+          "path": "file:///home/felix/scm/webknossos/binaryData/Organization_X/test-dataset/segmentation/1/",
```

Comment:
I think that won’t work, as not everyone has `/home/felix` – this might be a candidate for replaceVolatileValues.